lib/l10n/app_it.arb

```diff
@@ -267,15 +267,15 @@
   "change_cover": "Cambia copertina",
   "add_cover": "Aggiungi copertina",
   "restore_defaults": "Ripristina default",
-  "download_music_codec": "Codec musicale scaricamento",
-  "streaming_music_codec": "Codec musicale streaming",
-  "login_with_lastfm": "Accesso a Last.fm",
-  "connect": "Connetti",
-  "disconnect_lastfm": "Disconnetti Last.fm",
+  "download_music_codec": "Codec scaricamento musica",
```

Comment:
For consistency I would keep "download" as done previously

Reply:
I don't know, I just like "download" more than "scaricamento" but it's personal preference.

Reply:
Yeah. I see you changed "scaricamento" with "download" everywhere. I would keep "download" here, too. So, you can change the translation in "Codec download musica"

Reply:
Yeah I forgot that line. I'll change it soon.